Update READMEs

benmccann opened the pull request:
At first I was just going to remove this thinking that there was no need for it given the README in the main repo, but then I decided it'd be useful to have some minimal content here to help users who might want to contribute and are navigating the directory structure and also realized that it probably shows up on npm

Rich-Harris commented:
Given that it _does_ show up on npm (https://www.npmjs.com/package/@sveltejs/kit) I think it should probably be more fleshed out. Calling it a 'library' is probably a bit of a misnomer; at its heart it's a CLI. The only time you'd actually import `@sveltejs/kit` that I can think of is when you need to import types

benmccann commented:
I updated the description a bit based on your feedback. I also added a link to the work-in-progress docs. I'd hope not to duplicate too much from them and just be able to point people there

Conduitry commented on packages/kit/README.md:
```diff
@@ -1,15 +1,7 @@
-Clone this repo, `npm install`, and `npm link`. That will create a global link to the `svelte` bin. You can then either `npm run build` or `npm run dev`, if you intend to make changes and see them immediately reflected.
-
-Then, clone the corresponding [svelte-app-demo](https://github.com/sveltejs/svelte-app-demo) repo and follow the instructions therein.
+Please see [the documentation](https://kit.svelte.dev/docs) for information about getting started and developing with SvelteKit.
```
Is there a particular reason to point to this page rather than just to the root of the Kit site as a whole?

benmccann replied:
I debated that. It seemed like it'd save most users a click in getting to the docs. The landing page is just marketing and we don't really need users to see it first on their way to the docs

Conduitry commented on packages/kit/README.md:
```diff
@@ -1,15 +1,15 @@
+# The fastest way to build Svelte apps
 
-Here be dragons, etc etc.
+This is the [SvelteKit](https://kit.svelte.dev) framework and CLI. 
```
The trailing space is making linting fail.

If you're using VS Code and you have the appropriate extensions installed, then you _should_ already be getting format-on-save happening, and this sort of thing shouldn't be a problem.

hyunbinseo commented:
[npm](https://www.npmjs.com/package/@sveltejs/kit) is showing the older version of `packages/kit/README.md`.

As a result, link to [change-log](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) is not accessible in npm.

`Repository` and `Homepage` links should be set as well.

![image](https://user-images.githubusercontent.com/47051820/152898422-ab3c6f9c-e33a-41ce-b24e-3544d3ca9746.png)

Conduitry commented:
Yes. That will resolve itself once we start publishing as `@latest` on npm rather than just `@next`.
